### ISSUE TITLE
Crash while syncing RTC

### ISSUE THREAD

**Issue by noelvissers** [noelvissers/esp32-smart-clock#1]
[Time] Syncing with RTC...
CRtc RtcSync made...
Guru Meditation Error: Core  1 panic'ed (IntegerDivideByZero). Exception was unhandled.
Core 1 register dump:
PC      : 0x4015270b  PS      : 0x00060630  A0      : 0x80152bac  A1      : 0x3ffb1e40  
A2      : 0x00000133  A3      : 0x3ffb1ecc  A4      : 0x00000032  A5      : 0x3ffbf53c  
A6      : 0x00013880  A7      : 0x60013000  A8      : 0x00000113  A9      : 0x3ffb1e20
A10     : 0x00000000  A11     : 0x00000000  A12     : 0x3ffb450c  A13     : 0x3ffc14f4
A14     : 0x00002710  A15     : 0x3ffbf53c  SAR     : 0x0000001c  EXCCAUSE: 0x00000006
EXCVADDR: 0x00000000  LBEG    : 0x4000c46c  LEND    : 0x4000c477  LCOUNT  : 0x00000000

Backtrace: 0x4015270b:0x3ffb1e40 0x40152ba9:0x3ffb1e70 0x4014a841:0x3ffb1e90 0x4014a8f5:0x3ffb1ec0 0x4014a975:0x3ffb1ef0 0x400d53c2:0x3ffb1f10 0x400d3b86:0x3ffb1f40 0x400d2e7f:0x3ffb1f70 0x400d351f:0x3ffb1f90 0x400e4ca9:0x3ffb1fb0 0x40089541:0x3ffb1fd0

Rebooting...
ets Jun  8 2016 00:22:57

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0018,len:4
load:0x3fff001c,len:1044
load:0x40078000,len:8896
load:0x40080400,len:5828
entry 0x400806ac
[Status] Initializing...

**Comment by noelvissers** [comment]
Probably cause:
RTC update in both cores causes issue with RTC 

Check if this is the issue: Display temperature instead of time/date and see if error doesnt occur. 
Fix: If it is the multithreading causing this issue, set a bool when starting sync, and check this in the time/rtc/display class. At that point the whole multithreading is only used for get the data online, which causes the most delay in the first place, so it will (hopefully) still make sense to have it in another core.

```
85 time.cpp > 84
79 > set bool > wait ~20ms > check rtc sync > in sync > unset bool
                                            > not in sync > set time > unset bool
```

**Comment by noelvissers** [comment]
Fixed in @9ab6158
